pyout.py

```diff
@@ -22,9 +22,6 @@ class Tabular(object):
 
     Parameters
     ----------
-    data : iterable
-        Each element should be a dictionary where keys are the column
-        names and values are the data to write.
     columns : list of str
         Column names
     id_column : int, optional
```

Comment:
better to specify by name (or better even allowing for a combination of names) of columns. e.g. `id_columns="name"` or `id_columns=('first', 'last')`

Reply:
yeah.. not sure why I went with `int` there